Dashboard/navigation.py

```diff
@@ -21,22 +21,23 @@ def get_current_page_name() -> str | None:
 def make_sidebar() -> None:
     """Creates sidebar"""
     page_name = get_current_page_name()
-
     with st.sidebar:
-        st.logo("logo/hermes_icon.png")
+        st.logo("logo/hermes_logo.png")
```

Comment:
consider using the orange logo

Reply:
That's a good spot. I will create a new pull request to update the Icons.